### PR TITLE
Adds "Retrive AWS Region" TTP

### PR DESCRIPTION
**Pull request by w0rk3r**

### CODE REVIEW

File: discovery/3fe792b0-66eb-40fc-aea1-bd6de5a97083.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+id: 3fe792b0-66eb-40fc-aea1-bd6de5a97083
+name: Retrive AWS Region
```
Comment (khyberspache) [inline discovery/3fe792b0-66eb-40fc-aea1-bd6de5a97083.yml:2]: `Retrieve`

Comment (khyberspache) [reply discovery/3fe792b0-66eb-40fc-aea1-bd6de5a97083.yml:2]: Also, let's be more explicit. 

> Retrieve instance AWS region from metadata

File: discovery/3fe792b0-66eb-40fc-aea1-bd6de5a97083.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+description: Retrieve host region/localization using the instance metadata
```
Comment (khyberspache) [inline discovery/3fe792b0-66eb-40fc-aea1-bd6de5a97083.yml:3]: Expand the description a bit. Again we want to help new people understand what this is, what this does, and where to use it.

> AWS instances have a link-local address that return various types of metadata. This endpoint will return the region data associated with the current instance.

**Review by khyberspache — CHANGES_REQUESTED** [review]
Minor changes

File: discovery/3fe792b0-66eb-40fc-aea1-bd6de5a97083.yml (hunk 1 above)
Comment (w0rk3r) [reply discovery/3fe792b0-66eb-40fc-aea1-bd6de5a97083.yml]: Done

File: discovery/3fe792b0-66eb-40fc-aea1-bd6de5a97083.yml (hunk 2 above)
Comment (w0rk3r) [reply discovery/3fe792b0-66eb-40fc-aea1-bd6de5a97083.yml]: Done

**Pull request merged**